server/sql_migrations/20200602163951_create_maintenance_requests.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE
+IF NOT EXISTS public.nsl_maintenance
```

Comment:
Can we actually rename this `maintenance_requests`?

Reply:
renamed table to maintenance_requests

Reply:
I don't think you pushed your change?